### PR TITLE
Feature/api

### PR DESCRIPTION
**Pull request by DejanuTeodora**
## Description
Got all freaks from API and displayed them.Added skills and projects from API and used technologies instead skills.

## Related
https://trello.com/c/6b64rpSA

## Link to demo
https://www.youtube.com/watch?v=ZsEGtn663Fk

## Steps to Test or Reproduce

1. Install app
2. Open app

### CODE REVIEW

**Comment by DejanuTeodora** [comment]
[iOS - Get all freaks from API and show them in a list](https://trello.com/c/6b64rpSA/159-ios-get-all-freaks-from-api-and-show-them-in-a-list)

File: Shared/NetworkingLayer/GraphQL/Model/Query.graphql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+query GetAllFreaks {
```
Comment (vladstanescu94) [inline Shared/NetworkingLayer/GraphQL/Model/Query.graphql:1]: Maybe change the file name to reflect the query?

File: Shared/NetworkingLayer/GraphQL/Query.graphql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+query Freaks {
```
Comment (vladstanescu94) [inline Shared/NetworkingLayer/GraphQL/Query.graphql:1]: Do we still use this query?

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]

File: Shared/NetworkingLayer/GraphQL/Query.graphql (hunk 2 above)
Comment (DejanuTeodora) [reply Shared/NetworkingLayer/GraphQL/Query.graphql:1]: No, we are no longer using it.

**Review by vladstanescu94 — APPROVED** [review]

**Review by vladstanescu94 — APPROVED** [review]

**Review by rmoldo — APPROVED** [review]

**Review by rmoldo — APPROVED** [review]

**Review by vladstanescu94 — APPROVED** [review]

File: Shared/NetworkingLayer/GraphQL/Model/FreaksNode.swift [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        description = technologyItem.description
+    }
+}
+
+extension Project {
+    init(from projectItem: ProjectItem) {
+        id = projectItem.id
+        name = projectItem.name
+        description = projectItem.description
+        technologies = []
```
Comment (vladstanescu94) [inline Shared/NetworkingLayer/GraphQL/Model/FreaksNode.swift:35]: Need to change this to get the real data

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]

File: Shared/Features/Freaks/ViewModel/FreaksViewModel.swift [hunk 4]
```diff
@@ -10,8 +11,16 @@ final class FreaksViewModel: ObservableObject {
         getFreaks()
     }
 
+    func loadFreaks() {
+        Network.shared.getAllFreaks { [weak self] result, _ in
+            guard let self = self else { return }
+            self.freaks = result
+            self.allFreaks = result
```
Comment (RalucaMihaela) [inline Shared/Features/Freaks/ViewModel/FreaksViewModel.swift:18]: Why do we need two variables to store all freaks?

**Review by RalucaMihaela — CHANGES_REQUESTED** [review]

File: Shared/Features/Freaks/ViewModel/FreaksViewModel.swift (hunk 4 above)
Comment (DejanuTeodora) [reply Shared/Features/Freaks/ViewModel/FreaksViewModel.swift:18]: We need this for the filterFreaks() function.

**Review by RalucaMihaela — APPROVED** [review]

**Pull request merged**